tests/scripts/flake8.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+flake8 .
```

Comment:
NIT: if this script is only calling `flake8 .` what do you think to remove it and add the call in the above consumers?

Reply:
this way I can run the same script from docker or on the host.
If I move in the docker script then I cannot use the same script for both

Reply:
we could have in the consumer script the call to `flake8 .`, but still I'd need this script for running on the host and I have to maintain eventual changes in both place